client/src/components/core/ms-sorter/MsSorterPopover.vue

```diff
@@ -19,6 +19,9 @@
       class="sorter-list"
       id="sort-item-list"
     >
+      <ion-label class="sorter-list__title body-sm">
+        {{ $t('FoldersPage.sort.sortBy') }}
```

Comment:
Should move this translation string, we're in `core`, we shouldn't find a reference to `FoldersPage` here.